miso-web/src/main/webapp/scripts/list.js

```diff
@@ -258,6 +258,9 @@ ListUtils = (function() {
       }
     },
     render : {
+      archived : function(data, type, full) {
+        return data ? "🗄" : "";
```

Comment:
Since we have it tabbified by platform, I don't think more tabs is the answer. I'll limit these to just active ones.